### PR TITLE
Netplay Community Cheats

### PR DESCRIPTION
**Pull request by CrankySupertoon**
Mario party cheats

### CODE REVIEW

**Comment by JMC47** [comment]
You removed parts of the INI that you shouldn't have, including the gameID and other sections, even if they  weren't being used.

File: Data/Sys/GameSettings/GMPE01.ini [hunk 1]
```diff
@@ -1,104 +1,291 @@
-# GMPE01 - Mario Party 4
```
Comment (Techjar) [inline Data/Sys/GameSettings/GMPE01.ini]: Don't remove the header and boilerplate sections.

File: Data/Sys/GameSettings/GP5E01.ini [hunk 2]
```diff
@@ -1,380 +1,783 @@
-# GP5E01 - Mario Party 5
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP5E01.ini]: Ditto.

File: Data/Sys/GameSettings/GP7E01.ini [hunk 3]
```diff
@@ -1,19 +1,365 @@
-# GP7E01 - Mario Party 7
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP7E01.ini]: Ditto again.

File: Data/Sys/GameSettings/GP6E01.ini [hunk 4]
```diff
@@ -0,0 +1,159 @@
+[Gecko]
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP6E01.ini]: Header and boilerplate needs to be added to this file.

File: Data/Sys/GameSettings/RM8E01.ini [hunk 5]
```diff
@@ -0,0 +1,485 @@
+[Gecko]
```
Comment (Techjar) [inline Data/Sys/GameSettings/RM8E01.ini]: This one too.

**Comment by Techjar** [comment]
Should put Mario Party in the title somewhere.

**Comment by JosJuice** [comment]
It looks like you changed the line endings to CRLF. It should be LF.

(The one thing a lint for INI files would be useful, I guess...)

**Comment by CrankySupertoon** [comment]
It was there but when I loaded it in the emulator it removed it. I’ll re add it

File: Data/Sys/GameSettings/GP6E01.ini [hunk 6]
```diff
@@ -0,0 +1,166 @@
+#GP6E01 - Mario Party 6
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP6E01.ini]: Comment should start with a space, like so: `# GP6E01 - Mario Party 6`

File: Data/Sys/GameSettings/RM8E01.ini [hunk 7]
```diff
@@ -0,0 +1,491 @@
+#RM8E01 - Mario Party 8
```
Comment (Techjar) [inline Data/Sys/GameSettings/RM8E01.ini]: Same here.

File: Data/Sys/GameSettings/GP7E01.ini [hunk 8]
```diff
@@ -1,19 +1,367 @@
...
-$Infinite Mileage
-02510C6A 0000C350
-$Unlock All Minigames
-02291804 0005FFFF
-$Unlock all Souvieners
-002922F8 000001FF
-002922FD 000002FF
-$Unlock All Sounds
-002922FA 000001FF
+#GP7E01 - Mario Party 7
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP7E01.ini]: And here.

File: Data/Sys/GameSettings/GMPE01.ini [hunk 9]
```diff
@@ -1,104 +1,297 @@
...
-00000000 40000000
-
-$Press Z+R To Be On Turn 20
-0A1E67F0 00000030
-0018FCFC 00000014
-
-$Press Z+L To Be On Turn 1
-0A1E67F0 00000050
-0018FCFC 00000001
+#GMPE01 - Mario Party 4
```
Comment (Techjar) [inline Data/Sys/GameSettings/GMPE01.ini]: Yeah...

File: Data/Sys/GameSettings/GP5E01.ini [hunk 10]
```diff
@@ -1,380 +1,785 @@
...
-00000000 40000000
-$Press L+X: Player Stops Moving
-044E3396 08000000
-8A2885FE 00004440
-0022A07A 00000001
-0022A182 00000001
-0022A28A 00000001
-0022A392 00000001
-00000000 40000000
+#GP5E01 - Mario Party 5
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP5E01.ini]: Last one.

**Comment by Techjar** [comment]
This PR is not mergeable in this state. You have pushed incorrect line endings (CRLF instead of LF) as mentioned, and there is too much commit spam so it needs to be squashed.

**Comment by CrankySupertoon** [comment]
@Techjar I’ll squash this when I post a new one. My only question is what is CRLF and LF. The cheats load on my dolphin

**Comment by Techjar** [comment]
CRLF in Windows-style line endings, LF in Unix-style. All files in the repo must be Unix-style for compatiblity with everything that isn't Windows.

**Comment by Techjar** [comment]
Also you need not create a new PR, you can simply squash and force push this one.

**Pull request closed without merging**